NOTICE

```diff
@@ -0,0 +1,5 @@
+Exporter for mysql daemon
```

Comment:
For repos with the `prometheus` org we'd like to be consistent with NOTICE, AUTHORS.md, etc. files. Could you make this:

```
Exporter for MySQL daemon.
Copyright 2015 The Prometheus Authors
```

The best place for initial authorship attribution would be in the AUTHORS.md file, like in this: https://github.com/prometheus/node_exporter/blob/master/AUTHORS.md